libs/common/src/lib/ai-models/answer-generation/answer-generation.component.ts

```diff
@@ -58,8 +59,13 @@ export class AnswerGenerationComponent extends LearningConfigurationDirective {
     }),
   });
   currentGenerativeModel?: LearningConfigurationOption;
-  hasOwnKey = false;
+  userKeyToggle = new FormControl<boolean>(false);
+  required = this.translate.stream('kb.ai-models.common.required');
```

Comment:
why using a stream here? with `instant` you would not need `| async` in the template

Reply:
Fixed